src/main/java/org/opensearch/security/authtoken/jwt/JwtVendor.java

```diff
@@ -154,7 +158,8 @@ public String createJwt(String issuer, String subject, String audience, Integer
             throw new Exception("The expiration time should be a positive integer");
         }
 
-        // TODO: Should call preparelaims() if we need roles in claim;
+        String listOfRoles = String.join(",", roles);
```

Comment:
If roles is empty, will this throw an uncaught error?

Reply:
I just updated that.